README.md

```diff
@@ -1 +1,39 @@
-# example-osgi-portlet
\ No newline at end of file
+# Example OSGi portlet
+
+This application serves as a small example portlet that can be used in
+an OSGi-compliant ecosystem, in this case Liferay 7.
+
+## Prerequisites:
+
+- JDK11
+- Maven > 3
+- Internet connection to resolve dependencies
+
+
+## Local execution
+
+For development purposes, the application can be run as servlet in a
+local Jetty environment, without the need to set up and host a local
+OSGi environment.
+
+To run the application as servlet locally, check out the latest commit
+locally, and run the following command in your command line:
+
+```
+mvn clean -P no-liferay install jetty:run
```

Comment:
```suggestion
mvn clean jetty:run -P no-liferay
```

Reply:
At least with the current command its not working for me

Reply:
Furthermore I get an error telling me I am not logged in. Have you had the same error?

Reply:
This works for me and it also does not install the portlet Jar into my local maven repository.

```suggestion
mvn -P no-liferay clean package jetty:run
```

Reply:
> Furthermore I get an error telling me I am not logged in. Have you had the same error?

That is not an error, it is a feature:

```
// Get user of current request. Is null if user is not logged in
String user = VaadinService.getCurrentRequest().getRemoteUser();
log.debug("Page requested from: " + user);

// Create layout with error message if user is not logged in
if (user == null) {
   setContent(buildNotLoggedInLayout());
   return;
}
```

Maybe that part should be removed if the example shall be as minimalistic as possible.

Reply:
Shouldn't you be able to run without packaging it?
`mvn -P no-liferay clean jetty:run` should do the trick